google_cloud_compute/plugin.spec.yaml

```diff
@@ -1111,6 +1105,7 @@ actions:
         title: Firewall Name
         description: Name of the firewall rule to return
```

Comment:
I think this should be the name of the firewall to return, not the rule name?

and if it returns firewall information can we update the description and title since this is a major version bump already.

if this is accurate, perhaps:

```
title: Get Firewall Details
descripton: Retrieve details on a firewall including its rulesets
```